editor/editor_interface.h

```diff
@@ -162,6 +162,7 @@ class EditorInterface : public Object {
 	void reload_scene_from_path(const String &scene_path);
 
 	PackedStringArray get_open_scenes() const;
+	Array get_open_scenes_roots() const;
```

Comment:
TypedArray should be preferred when type is known.
```suggestion
	TypedArray<Node> get_open_scenes_roots() const;
```